packages/nav-frontend-etiketter/stories/chromatic.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import EtikettBase, {
+  EtikettInfo,
+  EtikettAdvarsel,
+  EtikettFokus,
+  EtikettSuksess,
+} from "../src/index";
+import { Meta } from "@storybook/react/types-6-0";
+
+export default {
+  title: "Etiketter/All",
+  component: EtikettBase,
+} as Meta;
+
+export const All = () => (
+  <>
+    <EtikettInfo> Info</EtikettInfo>
+    {"  "}
```

Comment:
Jeg foretrekker at vi bruker styling for å løse det du gjør med `{"  "}` og `<br />` i dag. Et wrapper-element med `display: grid` og vennene hans er et forslag.